packages/autocannon-ui-backend/services/autocannon.js

```diff
@@ -8,6 +8,10 @@ function execute(options, cb) {
     connections,
     pipelining,
     duration,
+    timeout,
+    title,
```

Comment:
what's the purpose of the title if we're not showing it when we display the results?

Reply:
The title is there to be used with [issue #58](https://github.com/nearform/autocannon-ui/issues/58)
Since I give to the backend the value I ask it back to later shown it.

That change goes only into that issue?

Reply:
understood, you can keep it then!